PWGHF/HFC/Tasks/femtoDreamPairTaskCharmHadrons.cxx

```diff
@@ -80,16 +80,16 @@ struct HfTaskCharmHadronsFemtoDream {
   Configurable<float> charmHadPromptBDTmax{"charmHadPromptBDTmax", 1., "Maximum prompt bdt score Charm Hadron (particle 2)"};
 
   /// General options
-  Configurable<float> fCPRdeltaEtaMax{"fCPRdeltaEtaMax", 0.01, "Max. Delta Eta for Close Pair Rejection"};
-  Configurable<float> fCPRdeltaPhiMax{"fCPRdeltaPhiMax", 0.01, "Max. Delta Phi for Close Pair Rejection"};
-  Configurable<bool> fCPRPlotPerRadii{"fCPRPlotPerRadii", false, "Plot CPR per radii"};
-  Configurable<bool> fExtendedPlots{"fExtendedPlots", false, "Enable additional three dimensional histogramms. High memory consumption. Use for debugging"};
-  Configurable<float> fHighkstarCut{"fHighkstarCut", 100000., "Set a cut for high k*, above which the pairs are rejected"};
-  Configurable<bool> fIsMc{"fIsMc", false, "Set true in the case of a MonteCarlo Run"};
-  Configurable<bool> fSmearingByOrigin{"fSmearingByOrigin", false, "Obtain the smearing matrix differential in the MC origin of particle 1 and particle 2. High memory consumption. Use with care!"};
-  Configurable<bool> fUse4D{"fUse4D", false, "Enable four dimensional histogramms (to be used only for analysis with high statistics): k* vs multiplicity vs multiplicity percentil vs mT"};
-  Configurable<bool> fUseCPR{"fUseCPR", false, "Close Pair Rejection"};
-  ConfigurableAxis fDummy{"fDummy", {1, 0, 1}, "fDummy axis"};
+  Configurable<float> CPRdeltaEtaMax{"CPRdeltaEtaMax", 0.01, "Max. Delta Eta for Close Pair Rejection"};
+  Configurable<float> CPRdeltaPhiMax{"CPRdeltaPhiMax", 0.01, "Max. Delta Phi for Close Pair Rejection"};
+  Configurable<bool> CPRPlotPerRadii{"CPRPlotPerRadii", false, "Plot CPR per radii"};
```

Comment:
Use lowerCamelCase for variables.

Reply:
I found this line while looking naming conventions
```
All names in C++ code follow camel case convention. This is the practice of writing compound words so that each word begins with a capital letter. No underscores are allowed. Acronyms are the only exception : they can be capitalized.
```